src/Core/Content/Media/Exception/FileTypeNotSupportedException.php

```diff
@@ -2,16 +2,24 @@
 
 namespace Shopware\Core\Content\Media\Exception;
 
+use Shopware\Core\Framework\Feature;
 use Shopware\Core\Framework\ShopwareHttpException;
 use Symfony\Component\HttpFoundation\Response;
 
 class FileTypeNotSupportedException extends ShopwareHttpException
 {
-    public function __construct(string $mediaId)
+    /**
+     * @deprecated tag:v6.5.0 - Parameter extension will be mandatory
+     */
+    public function __construct(string $mediaId, string $extension = '')
     {
+        Feature::triggerDeprecationOrThrow(
```

Comment:
shouldn't we check for am empty string first before triggering the deprecation warning?

Reply:
Sounds really legit 😂